README.adoc

```diff
@@ -1,35 +1,86 @@
-// Before publishing your new repository:
-// 1. Write the readme file
-// 2. Update the issues link in Contributing section in the readme file
-// 3. Update the discussion link in config.yml file in .github/ISSUE_TEMPLATE directory
+= Java Operating System Statistics (jos_01)
 
-= repo-template
-
-// Add a short description of your project. Tell what your project does and what it's used for.
-
-This is a template repository for Teragrep organization.
+This project will extract statistics from the operating system and running processes into Java objects.
+You can use this to easily get information such as memory usage about running processes and the operating system itself.
+The information provided is based on the Linux filesystem's /proc files.
 
 == Features
```

Comment:
This section seems like it should be an unordered list but it's not. Add one or rewrite the section in the way it's a consistent paragraph and not list of random sentences.